pkg/gitlab/mr.go

```diff
@@ -5,19 +5,17 @@ import (
 )
 
 func (client *Client) MRNumberWithSHA(owner, repo, sha string) (int, error) {
-	return 0, fmt.Errorf("not yet Supported MRNumberWithSHA method")
-	// prs, _, err := client.mr.ListMergeRequestsWithCommit(ctx, owner, repo, sha, &gitlab.MergeRequestListOptions{
-	// 	State: "all",
-	// 	Sort:  "updated",
-	// 	ListOptions: gitlab.ListOptions{
-	// 		PerPage: 1,
-	// 	},
-	// })
-	// if err != nil {
-	// 	return 0, fmt.Errorf("list associated merge requests: %w", err)
-	// }
-	// if len(prs) == 0 {
-	// 	return 0, errors.New("associated merge request isn't found")
-	// }
-	// return prs[0].GetNumber(), nil
+	mrList, _, err := client.commit.ListMergeRequestsByCommit(
+		fmt.Sprintf("%s/%s", owner, repo),
+		sha,
+	)
+	if err != nil {
+		return 0, err
```

Comment:
The test is failed below.
So, should be modified as shown in the Suggestions.
```suggestion
		return 0, fmt.Errorf("list associated pull requests: %w", err)
```